.github/workflows/release.yml

```diff
@@ -1,28 +1,31 @@
-name: Release
+name: CI/CD
 
 on:
   push:
     tags:
       - 'v*'
+  pull_request:
+    types: [ opened, synchronize ]
```

Comment:
Interesting! Didn't know about "synchronize" event on PRs before this!

Reply:
I still don't know much about it, I just blindly accepted what Aider suggested :)